app/controllers/rails_admin/main_controller.rb

```diff
@@ -39,6 +40,14 @@ def list_entries(model_config = @model_config, auth_scope_key = :index, addition
 
   private
 
+    def enforce_required_fields
```

Comment:
this is to alert user when they use empty filter directly in the url
such as:
http://localhost:3000/admin/pump?model_name=pump&utf8=%E2%9C%93&f%5Buser%5D%5B1%5D%5Bo%5D=like&f%5Buser%5D%5B1%5D%5Bv%5D=&query=

translate to something like: f: {user: { 1 => {o: 'like', v: '' } } } -> user contains ''